### ISSUE TITLE
Contribution 

### ISSUE THREAD

**Issue by anshkumar** [leohsuofnthu/Tensorflow-YOLACT#32]
Hi, 
I really liked your idea of implementing Yolact in tensorflow. I came around your repository around 4 months back. At that time, the repo was having a lot of bugs in it. (Like NaN loss, difference in training and validation losses etc.). I started working on those and currently have a working code in my repository (https://github.com/anshkumar/yolact). I would like to invite you to contribute to it, so that we can both move in right direction together.
